charts/k8ssandra/values.yaml

```diff
@@ -511,17 +532,25 @@ medusa:
 monitoring:
   grafana:
     # -- Enables the creation of configmaps containing Grafana dashboards. If
-    # leveraging the kube prometheus stack sub-chart this value should be
-    # `true`.
+    # leveraging the kube-prometheus-stack subchart this value should be
+    # `true`. See
+    # https://helm.sh/docs/chart_template_guide/subcharts_and_globals/ for
+    # background on subcharts.
     provision_dashboards: true
 
   prometheus:
     # -- Enables the creation of Prometheus Operator ServiceMonitor custom
-    # resources. If you are not using the kube prometheus stack sub-chart or do
+    # resources. If you are not using the kube-prometheus-stack subchart or do
     # not have the ServiceMonitor CRD installed on your cluster, set this value
     # to `false`.
     provision_service_monitors: true
 
+# -- The cleaner is a pre-delete hook that ensures the CassandraDatacenter
```

Comment:
I think after the CassandraBackup we could say it's a pre-delete hook that cleans up finalizers that will prevent the uninstallation. Not just cassdc.

Reply:
Not sure I understand. I reviewed the cleaner code and it just deletes the CassandraDatacenter that is installed with the release. What am I missing?

Reply:
The https://github.com/k8ssandra/medusa-operator/pull/26 will add a new finalizer, so the cleaner must be updated also to remove all these backups (or at least finalizers). Otherwise helm uninstall will no longer work. So it's going to be more common than just CassDC.

Reply:
How about something like:

 _The cleaner is a pre-delete hook that that ensures objects with finalizers get deleted. For example, cass-operator sets a finalizer on the CassandraDatacenter. Kubernetes blocks deletion of an object until all of its finalizers are cleared. In the case of the CassandraDatacenter object, cass-operator removes the finalizer. The problem is that there are no ordering guarantees with helm uninstall which means that the cass-operator deployment could be deleted before the CassandraDatacenter. The cleaner ensures that the CassandraDatacenter is deleted before cass-operator._

Reply:
Sounds good.